dask_ml/wrappers.py

```diff
@@ -202,13 +210,24 @@ def transform(self, X):
         """
         self._check_method("transform")
         X = self._check_array(X)
+        meta = self.meta
```

Comment:
Actually am curious whether we can just get `meta` from `X._meta`. We could also use [`meta_from_array`]( https://github.com/dask/dask/blob/ba78cda8167d78bed3f739b6e5d1504c2e99b46c/dask/array/utils.py#L23 )

cc @pentschev (in case you have thoughts here)

Reply:
This is a different meta than `X.meta` .  This is the meta we set here is for `predict`  and `transform` calls. 
See doc-string: 
https://github.com/dask/dask-ml/pull/862/files#diff-95e245a6cb7f05b184b66098b72f86099e09d2562205d9f22ed3f38298d89b46R157-R162

Reply:
For array-like, I would agree with @jakirkham suggestion for `meta_from_array`, but it seems like this is a more complex `meta` attribute that may combine either arrays or dataframes, then that wouldn't really apply here.

Reply:
The typical pattern to determine the new meta is to call the function on the old `.meta`. That's what `map_blocks` will do internally.

But we have an added complication that most scikit-learn functions will fail on length-0 inputs, so we'd need to generate some non-empty metadata.

Reply:
Yup. This attribute should work for  either arrays or dataframes.  Resolving this comment. 

Please feel free to re-open if you think we can make this cleaner. 👍🏼